ush/config_defaults.sh

```diff
@@ -2191,6 +2199,7 @@ SKEB_INT="3600" #Variable "skebint" in input.nml
 SKEBNORM="1"
 SKEB_VDOF="10"
 USE_ZMTNBLCK="FALSE"
+BLENDING_LENGTHSCALE="960" # (Lx) in kilometers
```

Comment:
Suggest changing to "`ENS_BLENDING_LENGTHSCAL`"

Reply:
done